desktop/package/linux/exchange.haveno.Haveno.metainfo.xml

```diff
@@ -3,6 +3,7 @@
   <id>exchange.haveno.Haveno</id>
   <!-- <icon type="stock">exchange.haveno.Haveno</icon> -->
   <launchable type="desktop-id">exchange.haveno.Haveno.desktop</launchable>
+  <launchable type="desktop-id">Haveno.desktop</launchable>
```

Comment:
Can one of these launchable configs be removed?

Reply:
Just removed the second one.